README.md

```diff
@@ -236,7 +213,12 @@ x-transportd:
   strip:
     # (int) Number of URL segments to remove from the beginning of the path before redirect.
     count: 0
-  headerinject:
+  requestheaderinject:
+    # ([]string) List of header names to inject.
+    names:
```

Comment:
suggestion: Now that @gcase555 has implemented the `map[string]string` configuration type, I think this would be much improved by using a map instead of two arrays. Something like:

```yaml
requestheaderinject:
  headers:
    x-header-1: "value"
    x-header-2: "value2"
```

Reply:
Yeah, agree!  I did not at all like the multiple list thing when I saw it but I just went with it.  I'll change the impl to your suggestion.

Reply:
FWIW, the impl is `map[string][]string`, which is in better alignment with HTTP spec's allowance of multiple headers of the same key anyway.  So the impl you see in this PR is actually
```
requestheaderinject:
  headers:
    x-header-1:
      - "value"
    x-header-2:
      - "value2"
```